docs/installation.rst

```diff
@@ -39,6 +39,7 @@ Add the following apps to the ``INSTALLED_APPS``:
         'django_otp.plugins.otp_static',
         'django_otp.plugins.otp_totp',
         'two_factor',
+        'two_factor.plugins.phonenumber',  # <- if you want phone number capability.
```

Comment:
I feel like this could be expanded upon. Maybe mention it with the the `PHONENUMBER_*` settings docs?